System.IO.Abstractions.TestingHelpers.Tests/MockFileSystemTests.cs

```diff
@@ -10,6 +10,22 @@ namespace System.IO.Abstractions.TestingHelpers.Tests
     [TestFixture]
     public class MockFileSystemTests
     {
+        [Test]
+        public void MockFileSystem_Constructor_ShouldAllowNullForEmptyFile()
```

Comment:
I don’t get what’s the difference between this test and the one below. Seems to be enough to keep the latter.

Reply:
I remove the constructor. After resolving conversation I'll try to find out how to update the PR.